### PR TITLE
[image] add Image loader for mobile (oneApp)

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/editor-common/web/src/Components/Loader.jsx [hunk 1]
```diff
@@ -109,6 +115,7 @@ Loader.propTypes = {
   theme: PropTypes.object.isRequired,
   isFastFakeLoader: PropTypes.bool,
   isVerySlowFakeLoader: PropTypes.bool,
+  manualPercent: PropTypes.number,
```
Comment (noamtwix) [inline packages/editor-common/web/src/Components/Loader.jsx:118]: manualPercent => percent

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,10 @@
 - `wrapper`
   - [#1132](https://github.com/wix-incubator/rich-content/pull/1132) palettes not working
 
+### :house: Internal
+- `image`
+  - [#1136](https://github.com/wix-incubator/rich-content/pull/1136) add image loader for mobile (oneApp)
```
Comment (shaulgo) [inline CHANGELOG.md:31]:
```suggestion
  - [#1136](https://github.com/wix-incubator/rich-content/pull/1136) add image loader for oneApp
```

File: packages/plugin-image/web/src/image-component.jsx [hunk 3]
```diff
@@ -123,8 +123,10 @@ class ImageComponent extends React.Component {
 
   handleCaptionChange = caption => this.handleMetadataChange({ caption });
 
-  renderLoader = () => {
-    return <Loader type={'medium'} isFastFakeLoader />;
+  renderLoader = (manualPercent = null) => {
+    return (
+      <Loader type={'medium'} manualPercent={manualPercent} isFastFakeLoader={!manualPercent} />
```
Comment (noamtwix) [inline packages/plugin-image/web/src/image-component.jsx:128]: remove isFastFakeLoader

File: packages/plugin-image/web/src/image-component.jsx [hunk 4]
```diff
@@ -123,8 +123,8 @@ class ImageComponent extends React.Component {
 
   handleCaptionChange = caption => this.handleMetadataChange({ caption });
 
-  renderLoader = () => {
-    return <Loader type={'medium'} isFastFakeLoader />;
+  renderLoader = (percent = null) => {
```
Comment (noamtwix) [inline packages/plugin-image/web/src/image-component.jsx:126]: remove 'null'

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 5]
```diff
@@ -220,12 +220,20 @@ class ImageViewer extends React.Component {
 
   handleContextMenu = e => this.props.disableRightClick && e.preventDefault();
 
+  // eslint-disable-next-line complexity
   render() {
     this.styles = this.styles || mergeStyles({ styles, theme: this.props.theme });
-    const { componentData, className, settings, setComponentUrl, seoMode } = this.props;
+    const {
```
Comment (shaulgo) [inline packages/plugin-image/web/src/image-viewer.jsx:226]: revert changes to this file. Use loader in image-component

File: packages/plugin-image/web/src/image-component.jsx (hunk 4 above)
Comment (shaulgo) [inline packages/plugin-image/web/src/image-component.jsx:126]:
```suggestion
  renderLoader = percent => {
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -44,6 +44,8 @@
 - `test-env`
   - [#1183](https://github.com/wix-incubator/rich-content/pull/1183) ricos coverage of file-upload + rename from wrapper
   - [#1188](https://github.com/wix-incubator/rich-content/pull/1188) fix link-preview `enableEmbed` on RicosTestApp
+- `image`
```
Comment (shaulgo) [inline CHANGELOG.md:47]: fix

File: packages/editor-common/web/src/Components/Loader.jsx [hunk 7]
```diff
@@ -48,6 +48,11 @@ class Loader extends React.Component {
   state = { percent: 1 };
 
   componentDidMount() {
+    const { percent } = this.props;
+    if (percent) {
```
Comment (shaulgo) [inline packages/editor-common/web/src/Components/Loader.jsx:52]: change to !percent and wrap the code

File: packages/editor-common/web/src/Components/Loader.jsx [hunk 8]
```diff
@@ -73,14 +78,15 @@ class Loader extends React.Component {
   }
 
   renderProgress() {
+    const { percent } = this.props;
```
Comment (shaulgo) [inline packages/editor-common/web/src/Components/Loader.jsx:81]: const percent = this.props.percent || this.state.percent

**Pull request merged**